1-Draft/RFCNNNN-Multi-Line-Continuance-For-Commands.md

```diff
@@ -102,13 +102,23 @@ That command will not parse because there is no closing brace for the script blo
 
 ### Inline splatting
 
-There has also been some discussion about the idea of inline splatting, using a format like `-@{...}` or `-@(...)`. Inline splatting has also been discussed separately on [RFC0002: Generalized Splatting](https://github.com/PowerShell/PowerShell-RFC/blob/master/2-Draft-Accepted/RFC0002-Generalized-Splatting.md), but using the syntax `@@{...}` or `@@(...)`. Using splatting to be able to span a single command across multiple lines has several limitations, including:
+There has also been some discussion about the idea of inline splatting, using a format like `-@{...}` or `-@(...)`. Inline splatting has also been discussed separately on [RFC0002: Generalized Splatting](https://github.com/PowerShell/PowerShell-RFC/blob/master/2-Draft-Accepted/RFC0002-Generalized-Splatting.md), but using the syntax `@@{...}` or `@@(...)`. Here is an example showing what that might look like:
+
+```PowerShell
+Get-ChildItem -@{
+    LiteralPath = $rootFolder
+    File = $true
+    Filter = '*.ps*1'
+}
+```
+
+Using inline splatting to be able to span a single command across multiple lines like this has several limitations, including:
 
 1. You cannot transition to/from the inline splatted syntax without a bunch of manual tweaks to the command (either converting parameter syntax into hashtable or array syntax or vice versa).
-1. You're forced to choose between named parameters or positional parameters/arguments. i.e. You can splat in a hashtable of named parameter/value pairs or an array of positional values, but you can't mix the two.
+1. You're forced to choose between named parameters or positional parameters/arguments. i.e. You can splat in a hashtable of named parameter/value pairs or an array of positional values, but you can't mix the two (the example shown just above is also used earlier in this RFC with positional parameters and switch parameters used without values, matching the way it is often used as a single-line command).
 1. There's no way to include unparsed arguments after the stop-parsing sigil in splatting.
```

Comment:
I don't see that either this point or the prior point are accurate.

Today you can have multiple splats in a single statement and you can also have positional parameters along with multiple splats. I don't see why that should change. No different than a "cuddled else", you simply have to start the next inline-splat (or stop parsing sigil) on the line the last inline splat completed.

```PowerShell
Get-ChildItem -@{
    LiteralPath = $rootFolder
    File = $true
    Filter = '*.ps*1'
} -switchparam -@{ 
  a=1
  b='2'
} --% a b 1 2
```

Reply:
1. Nobody said splatting itself should change.
1. You're right, you can splat multiple collections, and use the stop-parsing sigil at the end. I updated my wording.
1. The splatting example you've provided is quite different than a cuddled else.
1. Look at your syntax example again, and then say out loud "you simply have to start the next inline-splat...".

Are you really suggesting that going from something like this:

```PowerShell
Get-ChildItem $rootFolder -File -Filter *.ps*1
```

to this:

```PowerShell
Get-ChildItem -@(
    $rootFolder
) -@{
    File = $true
    Filter = '*.ps*1'
}
```

is simple? For someone who doesn't even necessarily work with hashtables? Or arrays?

How can that syntax, which is anything but simple, be a better alternative than this:

```PowerShell
Get-ChildItem @
    $rootFolder
    -File
    -Filer *.ps*1

```

The syntax you proposed would be possible with _multiple_ inline splatted collections, but as shown above it would also requiring the conversion of all parameters into multiple inline splatted collections, changing the syntax used for the parameters in the process (wrapping in quotes, adding `$true` for switch parameters, setting up the structure/values properly for arrays/hashtables (which means leaving command mode parsing and converting the values to work in expression-mode parsing, which is what is required for use in hashtables/arrays).

The intent of this RFC is to propose an _easier_ way to transition between single-line and multi-line commands...one that can easily be transitioned into or out of with minimal effort.

Reply:
Jesus Christ.

Listen to yourself.

You don't want to hear anything or anyone that disagrees with you. My prior decision was the correct one. I'll make no more comments.

Reply:
I'm sorry you feel I've slighted you in some way, but I've listened to all feedback on an RFC that hasn't even officially opened for comments yet according to the RFC process, yours included. If you don't want to make any more comments, that's your decision.

Most of the feedback has been constructive. It doesn't have to be in agreement to be constructive; however, some of the feedback has been destructive/antagonistic, and despite that, I've still read all of those comments as well and responded to them. Does that mean I should change the RFC according to each and every one of those comments? No. But just because I don't agree with your perspective on inline splatting, for reasons that I've identified despite previous, it doesn't mean I'm not listening. It certainly does not warrant being vulgar and unprofessional.

Reply:
I'm  quite good with what I've written. It wasn't vulgar or unprofessional. It certainly was expository, drawing attention to how you replied.

Your artificial constructs that ridiculously add complexity to what I actually wrote - good for you. If it makes you happy - that's fine. You are a fine example of why people don't get involved in this process (just as @jaykul tried to draw more commenters into the RFC today).

Reply:
I appreciate everyone who has provided feedback.  I can understand the passion around this, but let's be sure to adhere to the [Code of Conduct](https://opensource.microsoft.com/codeofconduct/).  Please focus on the technical merits and discussion and don't make it (or take it) personally.